app/Livewire/Explore/Index.php

```
@@ -0,0 +1,42 @@
+            ->whereNotNull('latitude')
+            ->whereNotNull('longitude')
+            ->with('trip')
+            ->get();
+    }
+
+    /**
+     * Render the component.
+     */
+    public function render(): View
+    {
+        return view('livewire.explore.index', [
```

PHPStan (2.2.7)

[error] Method App\Livewire\Explore\Index::render() should return Illuminate\View\View but returns Illuminate\Contracts\View\View.


(return.type)